packages/web-components/fast-foundation/src/anchor/anchor.template.ts

```diff
@@ -7,7 +7,7 @@ import { Anchor } from "./anchor";
  * @public
  */
 export const AnchorTemplate = html<Anchor>`
-    <template class="${x => x.appearance}">
+    <template>
```

Comment:
You don't need the <template> element if you're not adding any attributes to it.